vaadin-crud-flow-parent/vaadin-crud-flow/src/test/java/com/vaadin/flow/component/crud/CrudTest.java

```diff
@@ -29,9 +29,7 @@ public void itemAvailableInAllEvents() {
                 .addDeleteListener(e -> Assert.assertNotNull(e.getItem()));
         systemUnderTest.addEditListener(e -> Assert.assertNotNull(e.getItem()));
         systemUnderTest.addSaveListener(e -> Assert.assertNotNull(e.getItem()));
-
-        // Client side new should not come with an item
-        systemUnderTest.addNewListener(e -> Assert.assertNull(e.getItem()));
+        systemUnderTest.addNewListener(e -> Assert.assertNotNull(e.getItem()));
```

Comment:
I think we also want to make sure that this item in the new event is the same that is in the editor. If it would be a separate instance for some reason, then initializing it would not affect the editor.